modules/tidb/src/main/java/org/testcontainers/containers/TiDBContainer.java

```diff
@@ -0,0 +1,146 @@
+package org.testcontainers.containers;
+
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author richardnorth
+ */
+public class TiDBContainer extends JdbcDatabaseContainer<TiDBContainer> {
+
+    public static final String NAME = "tidb";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("pingcap/tidb");
+
+    @Deprecated
+    public static final String DEFAULT_TAG = "v6.1.0";
+
+    @Deprecated
+    public static final String IMAGE = DEFAULT_IMAGE_NAME.getUnversionedPart();
+
+    public static final String IMAGE_TAG = DEFAULT_TAG;
+
+    public static final Integer TIDB_PORT = 4000;
+
+    private static final int REST_API_PORT = 10080;
+
+    private String databaseName = "test";
+
+    private String username = "root";
+
+    private String password = "";
+
+    /**
+     * @deprecated use {@link TiDBContainer (DockerImageName)} instead
+     */
+    @Deprecated
```

Comment:
Shall we have a `@Deprecated` stuff in the initial release?

Reply:
I wrongly thought it was necessary. I'll delete this function.